internals/sendemail.py

```diff
@@ -118,3 +122,95 @@ def receive(bounce_message):
   message.check_initialized()
   if settings.SEND_EMAIL:
     message.send()
+
+
+def _extract_addrs(header_value):
+  """Given a message header value, return email address found there."""
+  friendly_addr_pairs = list(rfc822.AddressList(header_value))
+  return [addr for _friendly, addr in friendly_addr_pairs]
+
+
+def call_py3_task_handler(handler_path, task_dict):
+  """Request that our py3 code handle the rest of the work."""
+  handler_host = 'http://localhost:8080'
+  if settings.APP_ID == 'cr-status':
+    handler_host = 'https://chromestatus.com'
+  if settings.APP_ID == 'cr-status-staging':
+    handler_host = 'https://cr-status-staging.appspot.com'
+  handler_url = handler_host + handler_path
```

Comment:
What happens to a temporary deployment here? e.g. I want to test a deployment on a PR

Reply:
Inbound email is always routed to the currently live version.  To do anything with inbound email, we would need to make the new version be the live version on the site.  So, to test a change to inbound email, we would need to make the new version live on the staging server.   Since we would need to make it live, it is no loss to hard-code the live site URLs here.  

We won't be changing inbound email every week.  For those other weeks, we can continue to test the "rc" version on staging before making the new version live.

For just testing some UI change temporarily, with not intention of making it live, you can still do that by hitting the "rc" version: https://rc-dot-app-py3-dot-cr-status-staging.appspot.com/features.   Or, if you want to make it temporarily live on staging, that should be fine too.